roofit/roofitcore/src/RooGlobalFunc.cxx

```diff
@@ -80,6 +79,36 @@ RooCmdArg processFlatMap(const char *name, Func_t func, Detail::FlatMap<Key_t, V
    return container;
 }
 
+int interpretString(std::string const &s)
+{
+   return gInterpreter->ProcessLine(s.c_str());
+}
+
+Color_t interpretColorString(std::string const &color)
+{
+   using Map = std::unordered_map<std::string, Color_t>;
+   // Color dictionary to define matplotlib conventions
+   static Map colorMap{{"r", kRed},   {"b", kBlue},  {"g", kGreen},   {"y", kYellow},
+                       {"w", kWhite}, {"k", kBlack}, {"m", kMagenta}, {"c", kCyan}};
+   auto found = colorMap.find(color);
+   if (found != colorMap.end())
+      return found->second;
+   // Use interpreter if color was not matched in the color map
+   return gInterpreter->ProcessLine(color.c_str());
```

Comment:
I was hoping we could really remove the call to the interpreter altogether. Would it be an option to use [`TColor::GetColorByName`](https://github.com/root-project/root/blob/e85177414e921831578b8ea869b2212d7fe53a23/core/base/inc/TColor.h#L92) for example?